hazelcast-integration/kubernetes/samples/embedded/src/main/java/com/hazelcast/kubernetes/CommandController.java

```diff
@@ -1,7 +1,7 @@
 package com.hazelcast.kubernetes;
 
 import com.hazelcast.core.HazelcastInstance;
-import com.hazelcast.core.IMap;
+import com.hazelcast.map.IMap;
```

Comment:
How was that even compiling?

Reply:
I mean hazelcast-code-samples project.

Reply:
I think the Hazelcast dependency was redefined to 3.12 in `.../kubernetes/samples/embedded/pom.xml`. So, the parent dependency (Hazelcast 4) was not used.